### DISCUSSION TITLE
Card corners?

### DISCUSSION THREAD

**Discussion by mcdent** [piitaya/lovelace-mushroom#550] [General]
Hi, good work, thank you!
My cards don't seem to be showing the round corners as they should, however the chips seem ok, any ideas?
Thanks

![IMG_C5663DCCAD8E-1](https://user-images.githubusercontent.com/8769856/173202402-0b3a19ba-824a-488a-8fc6-c13d421a7d4a.jpeg)

**Comment by BVollmerhaus** [comment]
You'll need to install and select [Mushroom Themes](https://github.com/piitaya/lovelace-mushroom-themes) for the corners to be rounded as expected. This also styles non-Mushroom cards.

**Comment by Gitonthegrid** [comment]
You can also install the iOS themes, They look really nice with the Mushroom dashboards.
